### PR TITLE
[v2] Fix memory leak in processRequest for windows and darwin

### LINKED ISSUE #993: [v2] Windows, Darwin: Memory leak in processRequest
**Description**
Loading WebResources in processRequest have a memory leak for Darwin and Windows. Loading continuously big files will increase the memory usage and the memory is never freed.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a new v2 template
2. Add a huge HTML comment into index.html so the index.html get's several MB big, e.g. 3-4MB.
3. Add auto reload in the main.js.
```js
setTimeout(function() {
  window.location.reload();
}, 500)
```
4. Compile and run on windows or darwin
5. Monitor the memory usage of the process in the "Task-Manager" or "Activity Monitor".

**Expected behaviour**
Memory usage should not grow continuously and remain almost stable.

**System Details**
Wails Version: v2.0.0-beta.21
MacOS Version: 12.0.1
Go Version: go1.17.2
Windows Version: Windows 10 20H2

**Additional context**
For darwin the fix is pretty straight forward by releasing the NSData. For windows on the other hand, there are several instances which need to be released and the fix depends on fixes in `go-webview2`.
I'll provide a PR for `go-webview2` and `wails` shortly.

### PR DESCRIPTION
**Pull request by stffabi**
Fix #993 depends on leaanthony/go-webview2#4

### CODE REVIEW

**Comment by leaanthony** [comment]
@stffabi can you please join the Wails slack channel and message me. Thanks 🙏

File: v2/go.mod [hunk 1]
```diff
@@ -82,3 +82,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/go-webview2 v0.0.0-20211130085920-c044f547e45c => github.com/stffabi/go-webview2 v0.0.0-20211202081738-9524d55ebefc
```
[the remark this answers is not in the document]
Comment (leaanthony) [reply v2/go.mod:86]: Merged!!

**Comment by leaanthony** [comment]
This needs reading but we're good to go!

**Comment by stffabi** [comment]
Thanks, I've rebased the PR.

I've also added a new commit to remove the mod replace of #989, which got wrongly into the master.

**Comment by stffabi** [comment]
Thanks for merging.

**Pull request merged**